package.json

```diff
@@ -37,7 +37,8 @@
     "vaultclient": "scality/vaultclient",
     "werelogs": "scality/werelogs",
     "xml": "~1.0.0",
-    "xml2js": "~0.4.12"
+    "xml2js": "~0.4.12",
+    "node-getopt": "^0.2.3"
```

Comment:
There is already the module "commander" to handle CLI arguments, any particular reason to use this one ?